crates/cover_crypt/src/wasm_bindgen/tests.rs

```diff
@@ -44,10 +42,8 @@ fn encrypt_header(
         ),
         "Error encrypting header"
     );
-    EncryptedHeader::try_from_bytes(
-        &encrypted_header.to_vec()[CoverCryptX25519Aes256::SYM_KEY_LENGTH..],
-    )
-    .map_err(|e| JsValue::from_str(&e.to_string()))
+    EncryptedHeader::deserialize(&encrypted_header.to_vec()[Aes256Gcm::KEY_LENGTH..])
```

Comment:
The size of the key should depend on Covercrypt. It seems this size is not exposed. Please open an issue to track this.

Reply:
actually SYM_KEY_LENGTH is exposed